packages/zoe/src/contracts/multipoolAutoswap/pool.js

```diff
@@ -30,10 +30,10 @@ export const makeAddPool = (zcf, isSecondary, initPool, centralBrand) => {
       issuer: liquidityIssuer,
     } = liquidityZcfMint.getIssuerRecord();
 
-    const addLiquidityActual = (pool, userSeat, secondaryAmount) => {
+    const addLiquidityActual = (pool, zcfSeat, secondaryAmount) => {
```

Comment:
Nice! I'm not sure why this was called `userSeat`. Maybe to distinguish from `poolSeat`. Anyways, this is much better, thank you.